### PR TITLE
[INSTALLER] Removed thirdparties from here

### PR DESCRIPTION
**Pull request by jpfeuffer**
Adapted configuration template for this. Includes additional cleanups.

### CODE REVIEW

**Comment by jpfeuffer** [comment]
@timosachsenberg You included uptodate pwiz versions into the THIRDPARTY repository right?

**Comment by jpfeuffer** [comment]
I will test this on the build machines.

**Comment by timosachsenberg** [comment]
yep

File: OpenMS_installer.nsi [hunk 1]
```diff
@@ -21,8 +21,10 @@ Name "OpenMS"
 # set to "0" for deployment!!! use "1" to build the executable fast (for script debugging) 
 !define DEBUG_BUILD 0
 # set to "0" for deployment!!! use "1" to skip packaging of *.html files (takes ages)
-!define DEBUG_SKIP_DOCU 0
+!define DEBUG_SKIP_DOCU 1
```
Comment (jpfeuffer) [inline OpenMS_installer.nsi]: Note: Remind me to remove this before merging.

**Comment by timosachsenberg** [comment]
feel free to merge

**Comment by jpfeuffer** [comment]
Installer builds fine on the build machines. Can someone test the installer on a rather fresh Windows machine?
I can send/upload it.

**Comment by jpfeuffer** [comment]
Another question:
Currently I assume the structure of a complete clone of the THIRDPARTY repo for this script to work. However for packaging on Linux/Mac, we (historically) assume a "flattened" structure where only the matching subfolders are included e.g.:

```
Thirdparty
|- MSGFPlus
|- Fido
|- XTandem
...
```

instead of 

```
Thirdparty
|- All
   |- MSGFPlus
   |- Luciphor
|- MacOS
   |- 64bit
      |- Fido
...
```

I think we should focus on one style.
A complete clone is easier and straightforward, especially if you already checked it out but uses more space.

Alternative is a quick export (or git archive and unzip):

```
svn export --force https://github.com/OpenMS/THIRDPARTY/branches/master/All $WORKSPACE/SEARCHENGINES
svn export --force https://github.com/OpenMS/THIRDPARTY/branches/master/MacOS/$ARCH $WORKSPACE/SEARCHENGINES
```

Where you only have to be careful to download the right architecture for the build and is already used on the Unix build machines.

**Comment by timosachsenberg** [comment]
is the complete clone bad for path variable length?

**Comment by jpfeuffer** [comment]
Well, a bit worse. It adds e.g. Windows/64bit = 13 chars for every thirdparty engine binary that needs to be in the path.

**Comment by timosachsenberg** [comment]
hmm this is quite a lot considering that windows path lengths are still restricted to 260 characters. I think we should think about removing the dependency on using the PATH environment variables.

**Comment by jpfeuffer** [comment]
Ok I will adapt it to use a flattened folder structure. Although it needs a tiny bit more configuration effort it has several benefits nonetheless. E.g. saves traffic and space; easier traversal and handling in the scripts and consistency with our handling on Unix systems

**Pull request merged**